nselib/http.lua

```diff
@@ -101,6 +101,9 @@
 -- @args http.max-pipeline If set, it represents the number of outstanding  HTTP requests
 -- that should be pipelined. Defaults to <code>http.pipeline</code> (if set), or to what
 -- <code>getPipelineMax</code> function returns.
+-- @args http.cookiejar Parse a cookiejar and all the http requests in the script will have these cookies
```

Comment:
It would be great if we have two ways of defining cookies. One with structured tables as you have done so, and the other one using simple strings. Ex. --script-args http.cookiejar="SESSION=EAEAE". 
You can check internally what type of object the user passed with type().